controls/std_openeuler.yml

```diff
@@ -0,0 +1,1812 @@
+---
+policy: 'Standard Benchmark for openEuler'
```

Comment:
Does the policy have some resource? E.G. URL to the original document?

Reply:
yes, i have, i will paste it latter